### ISSUE TITLE
Paste as plain text

### ISSUE THREAD

**Issue by Romachamp** [p0deje/Maccy#858]
### Before Submitting Your Feature Request

- [X] Check that there isn't already a similar feature request to avoid creating a duplicate.
- [X] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Problem

Hello! I would like to be able to paste text as plain without any links, effects(bold, italic, etc), etc. This applies well to pasting from popular websites like Wikipedia. 

### Solution

Please, consider implementing this. 

Thank you, in advance!

**Comment by p0deje** [comment]
You can enable "Paste without formatting" in settings

**Comment by Romachamp** [comment]
Yes, but I want it to work in the background with any text, whenever I paste it.

**Comment by p0deje** [comment]
You can use Pure Paste app for this purpose.
